test/utils/datetime.test.js

```diff
@@ -210,4 +209,39 @@ describe('datetime', () => {
         .to.equal('14🙃20');
     });
   });
+
+  describe('formatDisplayDate', () => {
+    const tzAwareLA = {
+      timezoneAware: true,
+      timezoneName: 'America/Los_Angeles',
+    };
+    const tzAwareNY = {
+      timezoneAware: true,
+      timezoneName: 'America/New_York',
+    };
+    const tzUnaware = {
+      timezoneAware: false,
+      timezoneName: null,
+    };
+    const hammertime = 1473048000000; // 09/05/2016 @ 4:00am (UTC)
```

Comment:
Since the function now accepts either hammertime or a zulu timestamp string, we now set this from the ISO 8601 string which resides directly above it for clarity.